libs/datamodel/core/tests/base/duplicates.rs

```diff
@@ -37,13 +37,16 @@ fn fail_on_duplicate_models_with_map() {
         }
     "#;
 
-    let errors = parse_error(dml);
+    let expectation = expect![[r#"
+        [1;91merror[0m: [1mThe model with database name "User" could not be defined because another model with this name exists: "User"[0m
+          [1;94m-->[0m  [4mschema.prisma:5[0m
+        [1;94m   | [0m
+        [1;94m 4 | [0m
+        [1;94m 5 | [0m            @@[1;91mmap("User")[0m
+        [1;94m   | [0m
+    "#]];
```

Comment:
I'm not so sure do I like to write these in tests. They suck to read and if you have some nice way of generating them for copy&pasta, I'm all ears.

Reply:
This one is autogenerated (it's expect-test). The output is ugly in plain text but it's great in terminals because you see exactly what is highlighted. I like tests like this one a lot. Wdyt?

Reply:
It's ok, just that I can generate this somehow and if something changes, I don't need to spend lots of time modifying the strings...

Reply:
Yep that's what happens with these. snapshot tests are just perfect for that kind of tests. It's also something I want to discuss in the engines meeting tomorrow.